tgui/packages/tgui/layouts/Window.js

```diff
@@ -73,15 +73,10 @@ export class Window extends Component {
 }
 
 const WindowContent = props => {
-  const { scrollable, children } = props;
-  // A bit lazy to actually write styles for it,
-  // so we simply include a Box with margins.
+  const { scrollable, className, children } = props;
   return (
-    <Layout.Content
-      scrollable={scrollable}>
-      <Box m={1}>
-        {children}
-      </Box>
```

Comment:
I feel like this change will space a lot of existing UIs out too much, and I dont know how I feel about that

Reply:
Thing is, some components don't play nice with each other so it's better to give full control to the developer rather than have them do some workarounds to have a consistent experience